packages/core-components/src/layout/InfoCard/InfoCard.tsx

```diff
@@ -46,6 +46,10 @@ const useStyles = makeStyles(
     header: {
       padding: theme.spacing(2, 2, 2, 2.5),
     },
+    headerFixedContent: {
+      flexGrow: 1,
```

Comment:
Curious why the header component should always grow, instead of the content just aligning to the bottom for this variant. Suggestion bellow where separator would alway be directly under the header & the content would still align. 

<img width="537" alt="Screenshot 2024-03-22 at 11 45 03" src="https://github.com/backstage/backstage/assets/8904624/fb6fda69-0832-4ec6-9225-b276f59b1d73">

Reply:
This makes sense, and could even be useful for other `InfoCard`s. I added a commit which changes this, and replaces the new variant with a prop `alignContent` to be either `'normal'` (default) or `'bottom'`.